vendor/modules.txt

```diff
@@ -3064,7 +3065,7 @@ sigs.k8s.io/yaml
 # github.com/metal3-io/cluster-api-provider-baremetal => github.com/openshift/cluster-api-provider-baremetal v0.0.0-20190821174549-a2a477909c1d
 # github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20200929181438-91d71ef2122c
 # github.com/openshift/machine-config-operator => github.com/openshift/machine-config-operator v0.0.1-0.20201009041932-4fe8559913b8
-# github.com/terraform-providers/terraform-provider-aws => github.com/openshift/terraform-provider-aws v1.60.1-0.20210622193531-7d13cfbb1a8c
+# github.com/terraform-providers/terraform-provider-aws => github.com/mtulio/terraform-provider-aws v1.60.1-0.20211111005605-66e1a671ead5
```

Comment:
> `github.com/mtulio` 

Validation only, waiting for [PR](https://github.com/openshift/terraform-provider-aws/pull/10) review and tag creation to update here.